specutils/wcs/adapters/fitswcs_adapter.py

```diff
@@ -0,0 +1,31 @@
+from ..wcs_adapter import WCSAdapter, WCSAxes
+
+from astropy.wcs import WCS
+from astropy.wcs import WCSSUB_SPECTRAL, WCSSUB_LONGITUDE, WCSSUB_LATITUDE, \
+    WCSSUB_CUBEFACE, WCSSUB_STOKES, WCSSUB_CELESTIAL
+
+
+__all__ = ['FITSWCSAdapter']
+
+
+class FITSWCSAdapter(WCSAdapter):
+    wrapped_class = WCS
+    axes = None
+
```

Comment:
related to the `.system` discussion:

```
def __repr__(self):
      return "< WCS with axes {self.axes.names} and frames {self.ctype} >"
```
where 'axes.names' probably needs some work to infer, but `self.ctype` is almost exactly `self.wcs.ctype`